content/blog/2021/04/20-jenkins-contributor-awards.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Jenkins Contributor Awards - Nominations Open"
+tags:
+- awards
+- jenkins
+- cdcon
+author: cdfoundation
+description: "Nominate someone for the Jenkins Contributor Awards. Deadline is April 30."
+opengraph:
+  image: /images/post-images/2021/jenkins-awards-2021.png
+---
+This year, the Jenkins Contributor Awards will be run by the Continuous Delivery Foundation (CDF) along with many other link:https://cd.foundation/cdf-community-awards/[CDF Community Awards]. 
+
+Nominations are currently open for all three Jenkins awards:
```

Comment:
Briefly summarize a public nomination process as in https://twitter.com/oleg_nenashev/status/1384427322661642240 ?

Reply:
+1

Reply:
I can't seem to edit it again? You can use exactly what's in your tweet 
This year we will have public award nominations using GitHub issues. Any contributor is eligible this year. Nominate contributors or vote with reactions/comments!